airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/testing.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+
+from connector_ops.utils import Connector  # type: ignore
+from connectors_qa.models import Check, CheckCategory, CheckResult
+from pydash.collections import find
+
+
+class TestingCheck(Check):
+    category = CheckCategory.TESTING
+
+
+class IntegrationTestsEnabledCheck(TestingCheck):
+    name = "Medium to High Use Connectors must enable integration tests"
+    description = "Medium to High Use Connectors must enable integration tests via the `connectorTestSuitesOptions.suite:integrationTests` in their respective metadata.yaml file to ensure that the connector is working as expected."
+
+    def must_have_sandbox_config(self, connector: Connector) -> bool:
+        return connector.cloud_usage in ["medium", "high"]
+
+    def does_not_have_integration_enabled(self, connector: Connector) -> bool:
+        metadata = connector.metadata
+        connectorTestSuitesOptions = metadata.get("connectorTestSuitesOptions", [])
+        integrationTests = find(connectorTestSuitesOptions, {"suite": "integrationTests"})
+        return not integrationTests
```

Comment:
@alafanechere Just to confirm we only want to ensure theyve enabled the tests? and not to assert anything about secrets?

Reply:
Yes, we're not asserting anything about secrets.